src/modules/bricks/volume-bricks/volume-bricks.js

```diff
@@ -310,6 +335,17 @@
                 match = brick.status === filter.value.id || brick.status.toLowerCase() === filter.value.toLowerCase();
             } else if (filter.id === "devices") {
                 match = brick.devices[0].match(re) !== null;
+            } else if (filter.id === "utilMoreThan" || filter.id === "utilLessThan") {
+                percentage = parseInt(filter.value);
+                utilization = parseInt(brick.utilization.used);
+
+                if (percentage < 0 || percentage > 100) {
```

Comment:
indentation

Reply:
@cloudbehl As discussed, there is no indentation issue here.